src/components/Navigation.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  AppBar,
+  Toolbar,
+  Typography,
+  MenuItem,
+  Button,
+  Stack,
+} from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+
+export const Navigation = () => {
+  const { colors, shadow } = useTheme();
+
+  const appBarClass = {
+    backgroundColor: colors.white,
+    boxShadow: shadow.main,
+    padding: "0px 76px",
+  };
+
+  const menuItems = [" Locations", "My Places", "My Bookings"];
```

Comment:
This is a hard coded value and if it is always the same value (it is), you can extract it outside of this component or into another file (constants.ts)